http-delegation/src/main/scala/com/rackspace/httpdelegation/HttpDelegationHeader.scala

```diff
@@ -1,3 +1,22 @@
+/*
+ * #%L
```

Comment:
They are the open (#%L), section (%%), and close (#L%) markers used by the license-maven-plugin for identifying the headers it has generated.